app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
Right, but I believe API will definitely change so didn't want to do double job constantly updating method that nobody uses :)

But not a big deal, tell and I return it back :)